catalog.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="ru">
+   <head>
+      <meta charset="utf-8">
+      <title>HTML Academy: Седона</title>
+   </head>
+   <body>
+      <header>
+         <a href="index.html">
+         <img src="https://via.placeholder.com/138x70" alt="Логотип Седона">
+         </a>
+         <nav>
+            <ul>
+               <li><a href="#">Информация</a></li>
+               <li><a href="#">Фото и видео</a></li>
+               <li><a href="#">Карта штата</a></li>
+               <li><a>Гостиницы</a></li>
+            </ul>
+         </nav>
+      </header>
+      <main>
+         <section>
+            <h2>Фильтр отелей</h2>
+            <form action="https://echo.htmlacademy.ru" method="post">
+               <div>
+                  <p>Инфраструктура:</p>
+                  <input type="checkbox" name="pool" id="pool" checked>
+                  <label for="pool">
+                  "Бассейн"
+                  </label>
+                  <input type="checkbox" name="parking" id="parking">
+                  <label for="parking">
+                  "Парковка"
+                  </label>
+                  <input type="checkbox" name="wifi" id="wifi">
+                  <label for="wifi">
+                  "Wi-fi"
+                  </label>
```

Comment:
```suggestion
                  <p>Инфраструктура:</p>
                  <input type="checkbox" name="features" value="pool" id="pool" checked>
                  <label for="pool">
                  "Бассейн"
                  </label>
                  <input type="checkbox" name="features" value="parking" id="parking">
                  <label for="parking">
                  "Парковка"
                  </label>
                  <input type="checkbox" name="features" value="wifi" id="wifi">
                  <label for="wifi">
                  "Wi-fi"
                  </label>
```

Можно группировать чекбоксы по смыслу, задавая им одинаковое имя, так информацию будет легче ранжировать на сервере:

![image](https://user-images.githubusercontent.com/30810655/105980655-31b95800-60a6-11eb-88d9-3e899cd45f05.png)

_Это не замечание к исправлению, правь на свое усмотрение. Фомы будут позже в программе_